src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py

```diff
@@ -152,7 +152,8 @@ def timeseries(
             If True, all legends are drawn
             even if the number of legends exceeds the threshold.
         case : str
-            Parameter specifying best, worst or all. Use to create Response time timeseries graph.
+            Parameter specifying best, worst, worst_in_input, or all.
```

Comment:
```suggestion
            Parameter specifying best, worst, worst-in-input, or all.
```

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.